core/lib/libtomcrypt/src/ciphers/sub.mk

```diff
@@ -1,5 +1,13 @@
 cflags-y += -Wno-unused-parameter
 
+ifeq ($(CFG_CRYPTO_AES_ARM64_CE),y)
+srcs-y += aes_arm64_ce.c
+cflags-aes_arm64_ce.c-y += -march=armv8-a+crypto
```

Comment:
Are we guaranteed that the compiler will not emit neon instructions or anything like that even if this is specified?